### PR TITLE
S5t3 add contact api route

### PR DESCRIPTION
**Pull request by RachmannJoubert**

### CODE REVIEW

File: src/Classes/Controllers/AddContactController.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    /**
+     * AddContactController constructor.
+     * @param $hiringPartnerModel
+     */
+    public function __construct($hiringPartnerModel)
+    {
+        $this->hiringPartnerModel = $hiringPartnerModel;
+    }
+
+
```
Comment (mallowmew) [inline src/Classes/Controllers/AddContactController.php]: You need an invoke too - the method on the model you'll need to call is `addNewContact`. 
Look at `CreateHiringPartnerController` or `AddEventController` for examples.

File: src/Classes/Controllers/AddContactController.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            if (!empty($contact) && $contact instanceof HiringPartnerEntity) {
+                $result = $this->hiringPartnerModel->addNewContact($contact);
+            }
+        } catch (\Exception $exception) {
+            $data['message'] = $exception->getMessage();
+        }
+        if (isset($result) && $result) {
+            $data = [
+                'success' => true,
+                'message' => 'Hiring Partner Added to the db',
```
Comment (AlisonYoon) [inline src/Classes/Controllers/AddContactController.php]: Should probably say something like 'Hiring Parter Contact added to the db' ?

File: src/Classes/Controllers/AddContactController.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            $contact = $this->hiringPartnerModel->addNewContact(
+                $newContact['name'],
+                $newContact['email'],
+                $newContact['job_title'],
+                $newContact['phone'],
+                $newContact['hiring_partner_company_id'],
+                $newContact['is_primary_contact']
+            );
+            if (!empty($contact) && $contact instanceof HiringPartnerEntity) {
+                $result = $this->hiringPartnerModel->addNewContact($contact);
```
Comment (AlisonYoon) [inline src/Classes/Controllers/AddContactController.php]: I don't see addNewContact() in the hiringPartnerModel

Comment (RachmannJoubert) [reply src/Classes/Controllers/AddContactController.php]: The method will be created in a seperate task.

Comment (AlisonYoon) [reply src/Classes/Controllers/AddContactController.php]: nvm that's in another task

File: src/Classes/Controllers/AddContactController.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        $statusCode = 400;
+
+        try {
+            $contact = $this->hiringPartnerModel->addNewContact(
+                $newContact['name'],
+                $newContact['email'],
+                $newContact['job_title'],
+                $newContact['phone'],
+                $newContact['hiring_partner_company_id'],
+                $newContact['is_primary_contact']
```
Comment (mallowmew) [inline src/Classes/Controllers/AddContactController.php]: Can we have these fields named 

```
contactName
contactCompanyId
contactEmail
contactJobTitle
contactPhone
contactIsPrimary
```

**Pull request merged**